packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ModelController.java

```diff
@@ -86,25 +78,21 @@ ResponseEntity<Model> getModel(
 
 	@PutMapping("/{id}")
 	ResponseEntity<JsonNode> updateModel(
-		@PathVariable("id") String id,
-		@RequestBody Model model
-	) {
+			@PathVariable("id") String id,
+			@RequestBody Model model) {
 		return ResponseEntity.ok(proxy.updateModel(id, model).getBody());
 	}
 
 	@PostMapping
 	ResponseEntity<JsonNode> createModel(
-		@RequestBody Model model
-	) {
+			@RequestBody Model model) {
 		return ResponseEntity.ok(proxy.createModel(model).getBody());
 	}
 
-
 	@GetMapping("/{id}/model_configurations")
 	ResponseEntity<List<ModelConfiguration>> getModelConfigurations(
-		@PathVariable("id") String id,
-		@RequestParam("page_size") int pageSize
-	) {
+			@PathVariable("id") String id,
+			@RequestParam(value = "page_size", required = false, defaultValue = "100") int pageSize) {
 		return ResponseEntity.ok(proxy.getModelConfigurations(id, 100).getBody());
```

Comment:
Since we're asking for it, we may as well pass in pageSize rather than the hardcoded `100` here?